src/app/redux/ducks/busy/reducers.js

```diff
@@ -1,12 +1,18 @@
+function getMeta( meta ) {
```

Comment:
Please move the helper functions in a separate file (utils.js, as in the cart duck)